the-example-app/README.md

```diff
@@ -0,0 +1,22 @@
+What is `The Example App model`?
+================================
+
+The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
+Contentful. If you want to see the model in action, we do have a hosted version here: https://contentful-example-app.herokuapp.com/
+
+Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space. Doing so will be
+accomplished by this call to the command line app:
+
+```
+contentful space seed the-example-app
+```
+
+The app will then walk you through the needed steps in order to create a space based on it.
```

Comment:
Remove filler "then" and simplify: 

The app will walk you through the steps needed in order to create a space.